docs/docs/guides/FRAME_PROCESSORS_CREATE_OVERVIEW.mdx

```diff
@@ -51,8 +51,9 @@ Similar to a TurboModule, the Frame Processor Plugin Registry API automatically
 Return values will automatically be converted to JS values, assuming they are representable in the ["Types" table](#types). So the following Java Frame Processor Plugin:
 
 ```java
+@Nullable
 @Override
-public Object callback(Frame frame, Object[] params) {
+public Object callback(@NonNull Frame frame, @Nullable Map<String, Object> arguments) {
```

Comment:
Hmm I think leaving out the `@NonNull` and `@Nullable` props is easier to read for non-native developers, this is pretty verbose. Wdyt?

Reply:
I think that those decorators should stay. If anyone will be interacting with the docs, they will either copy-paste it to their IDE and will try to run it or they will try to compare it to what they have in their IDEs. Android Studio has quite good autocompletion and if somebody will try to generate it via Android Studio and then will see that docs are not referencing `@Nullable` or `@NonNull`, they will be confused regardless of their native development experience